src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure implements Drawable {
+    private double side;
+
+    public Square(String color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: "
+                + getArea() + " units, side: " + side
+                + "5 units, color: " + getColor());
```

Comment:
There seems to be a typo in the `draw` method output: `+ "5 units, color: " + getColor()`. The `5` should likely be removed to correctly display the side length and color.